pom.xml

```diff
@@ -19,7 +19,7 @@
         <java.version>1.8</java.version>
         <spring-boot-admin.version>2.1.3</spring-boot-admin.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <license.maven.plugin.version>4.0.rc2</license.maven.plugin.version>
+        <license.maven.plugin.version>4.0</license.maven.plugin.version>
```

Comment:
I applied the change and ran the "license:format" target; 37 files modified this year were updated to "2021".   The changes from "M2" are minimal; the change log is here: https://github.com/mycila/license-maven-plugin/milestone/16?closed=1

LGTM; Approved.

Reply:
Thanks!